content/en/docs/demo/services/reactnativeapp.md

```diff
@@ -0,0 +1,38 @@
+---
+title: React Native App
+cSpell:ignore: typeof
+---
+
+The React Native app provides a mobile UI for users on Android and iOS devices
+to interact with the demo's services. It is built with Expo and uses Expo's
```

Comment:
What is Expo?
We could add a link to the Expo explanation.

Reply:
added: https://github.com/open-telemetry/opentelemetry.io/pull/5704/commits/8304ec42659ada7c7d1cb77dfa43b1abac0a8c0d